cleanlab/experimental/tf_nlp.py

```diff
@@ -0,0 +1,129 @@
+"""
+A Scikeras classifier (adapted for huggingface models) which
+can be used for finding label issues in any text datasets.
+"""
+
+from typing import Dict
+from scikeras.wrappers import KerasClassifier
+from sklearn.preprocessing import FunctionTransformer
+import tensorflow as tf
+import numpy as np
+
+
+class HuggingfaceKerasClassifier(KerasClassifier):
+    def __init__(self, train_input: Dict, seq_len: int, **kwargs):
+        """
+        Basic scikeras/scikeras does not directly provide the option of having
+        a multi-input model, i.e the input must be in the form (num_samples, feature_size). However, you can work around this problem by
+        adding the `feature_encoder` property to the class which extends
+        Scikitlearn's BaseEstimator.
```

Comment:
Before the sentence starting with "However, ...", let's add one more sentence to explain what is "multi-input" about this model. Something like:

"In our case, the multiple inputs are: (1) token ids (input_ids); (2) indices specifying what tokens to attend to (attention_mask)".

Reply:
Also this sentence makes it sound like the user has to do this, but you've already nicely done it for them!  So I'd replace:
"However, you can work around this problem by adding the `feature_encoder` property to the class which extends Scikitlearn's BaseEstimator."
->
"Here we work around this by the `feature_encoder` property to our class which extends Scikit-learn's BaseEstimator"